.github/workflows/pr_agent.yml

```diff
@@ -0,0 +1,18 @@
+on:
+  pull_request:
+  issue_comment:
+jobs:
+  pr_agent_job:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+      contents: write
+    name: Run pr agent on every pull request, respond to user comments
+    steps:
+      - name: PR Agent action step
+        id: pragent
+        uses: Codium-ai/pr-agent@main
+        env:
+          OPENAI_KEY: ${{ secrets.OPENAI_KEY }}
+          GITHUB_TOKEN: ${{ secrets.GIGIGI }}
```

Comment:
It seems like the GITHUB_TOKEN is being set to a secret named GIGIGI. Make sure this secret exists and contains the correct value. [important]